### PR TITLE
feat(header): PRTL-136 login opens on enter

### PR DESCRIPTION
**Pull request by cy**
Tabbing over and pressing enter on Login, Download Token and Logout is the same as clicking

### CODE REVIEW

File: app/scripts/core/core.directives.ts [hunk 1]
```diff
@@ -69,6 +80,16 @@ module ngApp.core.directives {
...
+        };
+
+        $element.on('keypress', (e) => {
+          if (e.keyCode === KeyCode.Enter) {
+            openLogin();
+          }
+        });
+
+        $element.on('click', function(){
+          openLogin();
```
Comment (alex-wilmer) [inline app/scripts/core/core.directives.ts]: can this be `$element.on('click', openLogin)` ?

File: app/scripts/core/core.directives.ts [hunk 2]
```diff
@@ -94,8 +115,15 @@ module ngApp.core.directives {
...
-
           $window.location = redirect + authQuery;
+        };
+        $element.on('keypress', (e) => {
+          if (e.keyCode === KeyCode.Enter) {
+            logout();
+          }
+        });
+        $element.on('click',function() {
+          logout();
```
Comment (alex-wilmer) [inline app/scripts/core/core.directives.ts]: same as above

**Comment by alex-wilmer** [comment]
comments aside, 👍

File: app/scripts/core/core.directives.ts (hunk 1 above)
Comment (cy) [inline app/scripts/core/core.directives.ts]: good point

**Pull request merged**